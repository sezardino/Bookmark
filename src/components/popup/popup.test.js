import React from 'react';
import {shallow} from 'enzyme';

import Popup from './popup';

describe('Component Popup', () => {
	const component = shallow(<Popup />);

	it('Component Popup rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
