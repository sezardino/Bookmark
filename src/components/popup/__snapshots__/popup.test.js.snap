// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Popup Component Popup rendered correctly 1`] = `
"<div className=\\"popup \\">
  <div className=\\"popup__container\\">
    <button type=\\"button\\" className=\\"button popup__close\\" onClick={[Function: closePopup]}>
      <img src=\\"./img/popup-close.svg\\" alt=\\"close menu\\" />
    </button>
    <p className=\\"img-wrapper popup__img pseudo pseudo--left\\">
      <img src=\\"./img/illustration-features-tab-3.svg\\" alt=\\"some alternative text for seo\\" />
    </p>
    <div className=\\"popup__wrapper\\">
      <h2 className=\\"popup__title title\\">
        Share your bookmarks
      </h2>
      <p className=\\"popup__text text\\">
        Easily share your bookmarks and collections with others. Create a shareable link that you can send at the click of a button.
      </p>
      <Link to=\\"/info\\" className=\\"button button--popup button--blue\\" onClick={[Function: closePopup]}>
        More Info
      </Link>
    </div>
  </div>
</div>"
`;
